7/README.md

```diff
@@ -73,6 +74,9 @@ Key = DevString [link to RFC-9 Data Types]
 Value = An Arary of Any Tango compliant type or Pipe [link to RFC-9 Data Types]
```

Comment:
Value = An Arary ... => Value = An Array

Reply:
According to the doc, it seems like value can be a scalar of any tango compliant type too (https://tango-controls.readthedocs.io/en/latest/development/client-api/cpp-client-programmers-guide.html#reading-writing-device-pipe)

> In a DevicePipeBlob instance, you have:

    The blob name
    One array of DataElement. Each instance of this DataElement class has:
        A name
        A value which can be either
            Scalar or array of any basic Tango type
            Another DevicePipeBlob

Reply:
Oh! Thank you. My mistake. In Java it is array :)